frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/projects/ProjectPortlet.java

```diff
@@ -990,6 +1032,8 @@ private void prepareLicenseInfo(RenderRequest request, RenderResponse response)
                         filterAndSortAttachments(SW360Constants.LICENSE_INFO_ATTACHMENT_TYPES), true,
                         user);
                 request.setAttribute(PROJECT_LIST, mappedProjectLinks);
+                request.setAttribute(PortalConstants.PROJECT_RELEASE_RELATIONS, fetchReleaseRelationships(mappedProjectLinks));
+                request.setAttribute(PortalConstants.PROJECT_RELEASE_RELATIONS_EXCLUDED, fetchIcludedReleaseRelationships(id));
```

Comment:
Please change the constant name from PROJECT_RELEASE_RELATIONS_EXCLUDED to PROJECT_RELEASE_RELATIONS_INCLUDED

Reply:
Renamed